### DISCUSSION TITLE
Is Figma still a thing in Uno ?

### DISCUSSION THREAD

**Discussion by digger73** [unoplatform/uno#12547] [Q&A] [answered]
Hi

I'm considering using Figma with Uno for a rather large project. However, examining the current state of the Figma integration I'm actually reconsidering this, as the current state seems to be in preview and part of release 4.5.9 in september last year.
Furthermore, the documentation doesn't seem to be up to date as the link on https://platform.uno/docs/articles/figma-plugin/getting-started-uno-figma.html# to [Uno Material Toolkit Document](https://www.figma.com/community/file/1045411863379988189) results in a 404. 
One last thing is, that it isn't possible to find anything regarding a possible release instead of preview in the roadmap for Figma.

All of this makes me wonder if Figma has been abandoned or is it still a viable way for a large commercial projects, meaning is it going to be further worked on / released and maintained ?

**Comment by jeromelaban** [comment]
Thanks for the report. 

Could you tell use where you found this specific link? We'll need to adjust the source. To get the latest information, make sure to browse through our top level documentation https://platform.uno/docs/articles/intro.html, then select Figma from the left side menu.

It's very much maintained, and we're making regular updates to the tooling. Let us know if you find anything else that is blocking you!

**Reply by digger73** [reply]
Hi, 
Thanks for the quick reply.

It just doesn't seems maintained since nothing have been released specifically for Figma since september last year, which was a preview. Which could indicate a feature that is put on shelf and not maintained anymore. The same with the "Uno Material Toolkit Document" it was also last updated last year, so doesn't seem like any updates happening like you mention?
The Uno template in Figma is still designed for mobile and not cross platform as I see it ?

Following your description to the Figma description is the same link as I link to above: https://platform.uno/docs/articles/figma-plugin/getting-started-uno-figma.html# and in the section "Designer's Side->Quick Start" point 1, the link results in 404.

**Reply by jeromelaban** [reply]
I'll let @carldebilly and @NVLudwig comment on the Figma updates specifically.

With regards to the documentation, on the documentation's side (from https://platform.uno/docs/articles/intro.html), then [Figma/Overview](https://platform.uno/docs/articles/external/figma-docs/overview/getting-started-uno-figma.html), you should be heading to the [download page](https://platform.uno/docs/articles/external/figma-docs/download.html), which points to the appropriate Figma document. If you did not follow those steps, could you indicate which ones you did?

**Comment by carldebilly** [comment, marked as answer]
Hi @digger73, we have a team working on the plugin; it's definitely not something we've forgotten. You've found a link to an old documentation page that has since been updated (you can notice that there's nothing selected in the left-hand menu when navigating on this page).

We've already had a few releases this year and are actively working on another one (Preview 6). You can follow our updates here: https://platform.uno/blog/tag/figma/

We'll make sure that everything available on the documentation site is functioning well, and we thank you for reporting this!

Can you tell us more about important features you are expecting from the Figma experience ?

**Comment by digger73** [comment]
Hi @carldebilly  thanks for the clarification. Then we will continue on with the goal of using Figma for designing the UI. I can see that indeed I have hit an old doc. I didn't notice that the Figma part was running in its own "release" so I looked at the Uno releases and didn't see anything in the current releases of UNO and got worried.

**The current template / examples of Figma seems only to focus on mobile**, we are to also to develop for web, which seems not to be part of the examples. I'm fully aware that some mobile pages have to be designed different than web however sometimes simple scaling should be sufficient given the page. The main reason we are choosing Uno is because of the support for both, it would be nice with a simple example to show case how to handle both web and mobile in Figma, if this can be done in a smart way of course.

Currently we are getting to know what is available in Figma, it seems like for example simple data tables and graphs aren't available directly through Figma so it have to be handle afterwards in the XAML. Still in the learning process in regards to Figma as we are from WCF where everything was coded in XAML.

Thank you for the fast response :)

**Reply by carldebilly** [reply]
Indeed, the examples provided in the Figma document primarily illustrate a mobile version, but there's nothing in it or the generated code that prevents you from creating a version suitable for tablet, desktop, or even web. All you need to do is specify the appropriate resolution and it should work fine.

Figma is a flexible and powerful design tool that can accommodate any form factor or platform. You're not restricted to designing only for mobile. You can use Figma's layout grids and auto layout features to create designs that adapt to different screen sizes.

Regarding your concern about data tables and graphs, you're right, Figma doesn't directly provide these. But it's great for designing the look and feel of these elements. Once you have your design, you can then implement these features in your code using XAML and you can reference them as _external custom component_.

As you're coming from a WCF background where everything was coded in XAML, there might be a reduced learning curve. But I'm confident you'll find Figma plugin helpful once you get used to it. Its collaborative features and ability to quickly create and iterate on design ideas can significantly streamline the design process.


Which XAML controls do you consider to be the most important for your project?

Don't hesitate to ask if you have any more questions or need further clarifications. Happy designing with Figma and Uno!

**Reply by sasakrsmanovic** [reply]
One thing to note @digger73  is that we are coming up with some sample apps which will highlight multiple other resolutions and form factors so you can see it all in one single project. Stay tuned, we aren't that far from publishing it. Like Carl said, it would be great to learn what controls you need and to see what you are building. If you'd be interested in taking a 30 min call with our team please email me at sasha at platform dot uno
